executor/executor_bsd.h

```diff
@@ -56,13 +56,7 @@ static long execute_syscall(const call_t* c, long a[kMaxArgs])
 
 #if GOOS_freebsd
 
-#define KIOENABLE _IOW('c', 2, int) // Enable coverage recording
-#define KIODISABLE _IO('c', 3) // Disable coverage recording
-#define KIOSETBUFSIZE _IOW('c', 4, unsigned int) // Set the buffer size
-
-#define KCOV_MODE_NONE -1
-#define KCOV_MODE_TRACE_PC 0
-#define KCOV_MODE_TRACE_CMP 1
+#include <sys/kcov.h>
```

Comment:
I guess this will break building on linux, which gives us very valuable test coverage (at least basic build testing). If we do this freebsd build will break right tomorrow.
Let's leave it open-coded, it's not much code and since it's a public interface it should become stable once the patch is submitted.

Reply:
I see. Well, I think it is a matter of time, when building freebsd executor on linux will break. 
Waiting on patch submit is a good idea.

Reply:
Hard to say.
So far we don't see high rate of freebsd-specific changes, but there is large flow of linux-specific and generic changes that tend to break code that is not even built.
Alternatively, we could finish syzbot integration then syzbot will provide actual end-to-end freebsd testing. But nobody volunteered to do it so far.

Reply:
Another note is that this seems to require using the latest freebsd with the kcov patch as host/build machine, which I guess is not always true. I.e. testing the latest kernel, but using something older and more stable as host.